staging/src/k8s.io/client-go/discovery/discovery_client_test.go

```diff
@@ -639,7 +640,7 @@ func TestServerPreferredResourcesRetries(t *testing.T) {
 						{
 							Name: "extensions",
 							Versions: []metav1.GroupVersionForDiscovery{
-								{GroupVersion: "extensions/v1beta1"},
+								{GroupVersion: "extensions/v1beta1", Version: "v1beta1"},
```

Comment:
Please ensure that at least some unit test runs multiple groups and/or versions, otherwise the concurrency isn't really tested.

Reply:
TestServerPreferredNamespacedResources does (and was flaking on my original PR which was why I noticed)

Also added an additional version to TestGetServerResources and added order-sensitive checks to the test